### PR TITLE
Add Jdk-9 using Automatic module

### PR DESCRIPTION
**Pull request by Raghav2211**

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @rusty-neo,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by Raghav2211** [comment]
> Thanks for noticing this is missing!
> 
> The modules from akka itself have names like 'akka.actor', 'akka.cluster.metrics' and 'akka.stream.testkit'. Perhaps we should make this 'akka.stream.contrib'? Even though this library is not in the akka repository, it is still in the akka organization, so that might be more consistent?

Yes this is consistent for now because **akka-stream-contrib**  is packaging separately and in akka repo there is class called **AutomaticModuleName** which is converting all aka modules to automatic modules, when this library merge in akka then it will be build as a akka module and will be take care of their's  build.sbt.
So it think this is consistent for now in next release

**Comment by raboof** [comment]
We also put alpakka-kafka under the `akka.stream.` prefix though (https://github.com/akka/alpakka-kafka/blob/master/build.sbt#L134), and for that (like contrib) we also don't have any plans to move them into the Akka repo. What are the implications of `Automatic-Module-Name`? Are there any implications to the structure of the name chosen?

**Comment by Raghav2211** [comment]
> We also put alpakka-kafka under the `akka.stream.` prefix though (https://github.com/akka/alpakka-kafka/blob/master/build.sbt#L134), and for that (like contrib) we also don't have any plans to move them into the Akka repo. What are the implications of `Automatic-Module-Name`? Are there any implications to the structure of the name chosen?

No implication if you add any library, this change will only able to add this module in module-info.java where we need.

File: project/Common.scala [hunk 1]
```diff
@@ -24,7 +25,8 @@ object Common extends AutoPlugin {
   override lazy val projectSettings = SbtScalariform.scalariformSettings ++ Seq(
     organization := "com.typesafe.akka",
     organizationName := "Lightbend Inc.",
-
+    packageOptions := Seq(ManifestAttributes(
+                      ("Automatic-Module-Name", "akka-stream-contrib"))),
```
Comment (ennru) [inline project/Common.scala]: I think we'd like to go with `akka.stream.alpakka.contrib` instead.

**Comment by raboof** [comment]
(closing due to inactivity, feel free to reopen when work resumes!)

**Pull request closed without merging**